Bundle/CoreBundle/Resources/script/linkModal.js

```diff
@@ -5,9 +5,12 @@ $(document).ready(function() {
         $('.modal').modal('hide').on('hidden.bs.modal', function(){$(this).remove()});
 
         var url = $(this).attr('href');
+        $('#canvasloader-container').fadeIn();
```

Comment:
$('#canvasloader-container') doesn't exist in victoire repository but it can be useful.